.gitignore

```diff
@@ -7,6 +7,7 @@ docs/source/tutorials/package/*
 docs/source/tutorials/platform/*
 docs/source/tutorials_source/platform/data
 docs/source/tutorials_source/platform/pizzas
+docs/source/tutorials_source/platform/__MACOSX
```

Comment:
what is this?

Reply:
This is a folder with thumbnails from OSX. We already have a line for another folder. I'll combine the two.